### DISCUSSION TITLE
UptimeRobot showing monitor URLs

### DISCUSSION THREAD

**Discussion by cloneofghosts** [Pirate-Weather/pirateweather#146] [General]
@alexander0042 I noticed when visiting the status page that UptimeRobot is now showing the monitor URLs. I assume the key you are using is a test key but it's still not ideal. I've found that changing the density to normal fixes the issue.

**Comment by alexander0042** [comment]
Great catch here- they are pretty limited API keys (I set it so it would equal the number of status requests), but as a general practise, not a great plan. I switched things over to a new status page that I like netter anyway (https://pirateweather.xitoring.io/), so that should fix it!

**Reply by cloneofghosts** [reply]
Yeah, I figured the keys were limited but still not ideal to have them out in the open. I saw you updated the site with the new URL and I'll update the readme on the HA repository to have the new link.

The only thing the new site doesn't seem to show the response time like UptimeRobot but it's still not a huge deal.
